smriprep/workflows/outputs.py

```diff
@@ -538,44 +571,44 @@ def init_ds_template_registration_wf(
         name='outputnode',
     )
 
-    ds_std2t1w_xfm = pe.MapNode(
+    ds_std2anat_xfm = pe.MapNode(
         DerivativesDataSink(
             base_directory=output_dir,
-            to='T1w',
+            to=image_type,
             mode='image',
             suffix='xfm',
             dismiss_entities=['desc'],
         ),
         iterfield=('in_file', 'from'),
-        name='ds_std2t1w_xfm',
+        name='ds_std2anat_xfm',
         run_without_submitting=True,
     )
 
-    ds_t1w2std_xfm = pe.MapNode(
+    ds_anat2std_xfm = pe.MapNode(
         DerivativesDataSink(
             base_directory=output_dir,
             mode='image',
             suffix='xfm',
             dismiss_entities=['desc'],
-            **{'from': 'T1w'},
+            **{'from': image_type},
```

Comment:
We should think about whether the subject-native space should continue to be called `T1w`. We could call it `anat`, so we'd have `from-anat_to-MNI_xfm.h5` and `space-anat_desc-preproc_bold.nii.gz`.

But that does not need to be decided now, and it's better not to change the output names until we're sure that's what we want.

Reply:
I like calling it `anat`, but I feel it still carries some ambiguity. Perhaps having some metadata (either in the sidecar or a separate, generated file) covering exactly this space is (image dimensions, files used to derive it, etc) may be helpful.